CHANGELOG.rst

```diff
@@ -18,7 +18,8 @@ Added
 - Added listener for ``kytos/core.interface_tags`` event to save any changes made to ``Interface`` attributes ``tag_ranges`` and ``available_tags``
 - Added script ``special_vlan_allocation.py`` to add ``special_available_tags`` and ``special_tags`` fields to ``interface_details`` collection.
 - Added endpoint ``POST v3/interfaces/{interface_id}/special_tags`` to set ``special_tags`` to interfaces.
-- Added endpoint ``DELETE v3/links/{link_id}`` to delete a disabled link.
+- Added endpoint ``DELETE v3/links/{link_id}`` to delete a disabled link. To avoid rediscovery, the link needs to be physically disconnected or both interface ends (switches) need to be disabled.
+- Added endpoint ``DELETE /v3/switches/{dpid}`` to delete a disabled switch. This endpoint is for decommissioned switches. The switch will get rediscovered if it reconnects to the controller. So, after deleting a switch on ``topology``, you're expected to also remove the TCP connection configuration on the switch.
```

Comment:
It'd be great to also include in the changelog the new breaking change that now links need to be enabled explicitly again if one of its interfaces gets disabled. 

I'll also recommend that you broadcast this change on the Slack `#general` channel just so network engineers are aware about it (in the future they'll read the changelog), but if they have something to argument against it with any other strong argument then that's still in time.

Reply:
Updated changelog